src/common/args.cpp

```diff
@@ -717,14 +718,24 @@ fs::path ArgsManager::GetConfigFilePath() const
     return GetConfigFile(*this, GetPathArg("-conf", BITCOIN_CONF_FILENAME));
 }
 
-std::string ArgsManager::GetChainName() const
+ChainType ArgsManager::GetChainType() const
```

Comment:
In commit "refactor: Replace string chain name constants with ChainTypes" (ca13faf0cc8acbaf425ac69c783415465403b5a3)

Could this new method be added in an earlier commit? This commit is very long and all the other changes in the commit are mechanical changes except this one method, so the new code seems to get lost.

(This would also open the door to automating the other changes in this commit with a scripted-diff, though that would still be awkward not and not necessary IMO)